### DISCUSSION TITLE
Three fourths/First Two Thirds - Center

### DISCUSSION THREAD

**Discussion by ghost** [rxhanson/RectanglePro-Community#14] [Ideas]
Hi 👋🏻

On my side, a missing window position will be the **three fourths/first two thirds but in center** or something like the "Dock size".

Thanks a lot for your good job, I was able to abandon Moon.

**Comment by rxhanson** [comment]
I don't mind adding this in when I get a chance, but I've got a lot of features that are lined up to go in before that and these can be added in via the custom shortcut builder.
<img width="891" alt="Screen Shot 2022-02-08 at 9 55 25 PM" src="https://user-images.githubusercontent.com/13651296/153113391-f99dcd5e-da41-46b1-9fe6-e2a60b12f333.png">

Custom shortcuts can be selected in window throw / long throw at the end of their respective dropdowns, if you're looking to configure them there.

**Reply by ghost** [reply]
Thanks! I'm a new Rectangle Pro user :-)
